src/EloquentQueryModifier.php

```diff
@@ -652,6 +653,18 @@ protected function applyNestedJoins(array $relations, Model $instance, $field, $
                 // Join related's table on the base table's foreign key
                 $query->join($table, $instance->getQualifiedKeyName(), '=', $related->getQualifiedForeignKeyName());
                 break;
+            case HasManyThrough::class:
+                Relation::noConstraints(function () use ($related, $relation, $instance, $query, $field) {
+                    //Use the relationship to build a new query without the model constraint and name the subtable the same as the requested relationship name
+                    //Alias the "first" key to prevent collisions with other columns
+                    $subQuery = $instance->$relation()->select([
+                        $related->getQualifiedFirstKeyName().' AS '.($uniqeFirstKey = str_replace('.', '_', uniqId().$related->getQualifiedFirstKeyName())), //Id to left join on
```

Comment:
Minor spelling suggestion
```suggestion
                        $related->getQualifiedFirstKeyName().' AS '.($uniqueFirstKey = str_replace('.', '_', uniqueId().$related->getQualifiedFirstKeyName())), //Id to left join on
```